(docs) Add a defintion of 'slug' to Part 7 tutorial (#21939)
gatsbyjs/gatsby: docs/tutorial/part-seven/index.md

## Creating slugs for pages

A ‘slug’ is the unique identifying part of a web address,

> such as the `/tutorial/part-seven` part of the page https://www.gatsbyjs.org/tutorial/part-seven/.

Reviewer: ```suggestion
such as the `/tutorial/part-seven` part of the page `https://www.gatsbyjs.org/tutorial/part-seven/`.
```

> It is also referred to as the ‘path’ but we will use the term ‘slug’ in this tutorial.

Reviewer: ```suggestion
It is also referred to as the ‘path’ but this tutorial will use the term ‘slug’ for consistency.
```
